### PR TITLE
K8SPSMDB-441 mimic pxc doc about recovering from s3 to a new environment

### PR DESCRIPTION
**Pull request by fiowro**
[K8SPSMDB-441](https://jira.percona.com/browse/K8SPSMDB-441) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

### CODE REVIEW

**Review by qJkee — DISMISSED** [review]

**Review by qJkee — APPROVED** [review]

File: source/backups.rst [hunk 1]
```diff
@@ -172,54 +180,91 @@ message in the operator logs.
...
+restoration can be done in the following way.
+
+1. Set appropriate keys in the `deploy/backup/restore.yaml <https://github.com/percona/percona-server-mongodb-operator/blob/main/deploy/backup/restore.yaml>`_ file.
+
+   * set ``spec.clusterName`` key to the name of the target cluster to restore
+     the backup on,
+   * if you are restoring backup on the *same* Kubernetes-based cluster you have
+      used to save this backup, set ``spec.backupName`` key to the name of your
+      backup,
+   * if you are restoring backup on the Kubernetes-based cluster *different*
```
Comment (egegunes) [inline source/backups.rst:230]: It's also possible to use one of the storages in `cr.spec.backup.storages`:

```yaml
apiVersion: psmdb.percona.com/v1
kind: PerconaServerMongoDBRestore
metadata:
  name: restore1
spec:
  clusterName: my-cluster-name
  storageName: gcs
  backupSource:
    destination: s3://S3-BUCKET-NAME/BACKUP-NAME
```

**Review by qJkee — APPROVED** [review]

File: source/backups.rst [hunk 2]
```diff
@@ -172,54 +180,100 @@ message in the operator logs.
...
+            endpointUrl: https://URL-OF-THE-S3-COMPATIBLE-STORAGE
+   * you can also use a ``storageName`` key to specify the exact name of the
+     storage (the actual storage should be already defined in the
+     ``backup.storages`` subsection of the ``deploy/cr.yaml`` file):
+
+     .. code-block:: yaml
+
+        ...
+        backupSource:
+          storageName: s3-us-west
```
Comment (spron-in) [inline source/backups.rst:255]: Where do I specify the backup name in this case?

**Review by spron-in — CHANGES_REQUESTED** [review]
Please see the comments.
Also, for PITR - can we have an example with recovering with PITR from S3 bucket, not from existing backup CR?

File: source/backups.rst (hunk 2 above)
Comment (egegunes) [inline source/backups.rst:255]: @fiowro `storageName` field is under the `spec`, not `spec.backupSource`:

```yaml
apiVersion: psmdb.percona.com/v1
kind: PerconaServerMongoDBRestore
metadata:
  name: restore1
spec:
  clusterName: my-cluster-name
  storageName: gcs
  backupSource:
    destination: s3://S3-BUCKET-NAME/BACKUP-NAME
```

Comment (egegunes) [reply source/backups.rst:255]: @fiowro 

```yaml
  ...
  backupSource:
    destination: s3://S3-BUCKET-NAME/BACKUP-NAME
```

**Review by egegunes — APPROVED** [review]

**Review by qJkee — DISMISSED** [review]

**Review by egegunes — APPROVED** [review]

**Review by qJkee — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**